ansible/roles/openshift_outbound_tcp_logging/tasks/main.yml

```diff
@@ -43,42 +43,48 @@
       after: "id_outbound_tcp_log_4_"
       regex: "id_outbound_tcp_log_5_"
 
-    forward_logging_rules:
+    filtering_rules:
     # Matches first packet in new outbound TCP connections, -A for use in /etc/sysconfig/iptables.
-    - chain: FORWARD
-      line:  "-A FORWARD -o eth0 -p tcp ! -d 172.16.0.0/12 -m state --state NEW -m comment --comment id_outbound_tcp_log_7_ -j oo-outbound-logging"
+    - chain: OPENSHIFT-OUTBOUND-FILTERING
+      line:  "-A OPENSHIFT-OUTBOUND-FILTERING -o eth0 -p tcp ! -d 172.16.0.0/12 -m state --state NEW -m comment --comment id_outbound_tcp_log_7_ -j oo-outbound-logging"
       after: "^-A INPUT"
       regex: "id_outbound_tcp_log_7_"
 
-    forward_logging_memory:
+    filtering_memory:
     # Matches first packet in new outbound TCP connections, -I to send to the top of the FORWARD chain in memory.
-    - chain: FORWARD
-      line:  "-I FORWARD -o eth0 -p tcp ! -d 172.16.0.0/12 -m state --state NEW -m comment --comment id_outbound_tcp_log_7_ -j oo-outbound-logging"
+    - chain: OPENSHIFT-OUTBOUND-FILTERING
+      line:  "-I OPENSHIFT-OUTBOUND-FILTERING -o eth0 -p tcp ! -d 172.16.0.0/12 -m state --state NEW -m comment --comment id_outbound_tcp_log_7_ -j oo-outbound-logging"
       after: "^-A INPUT"
       regex: "id_outbound_tcp_log_7_" 
 
-- name: list iptables rules FORWARD chain
-  command: '/usr/sbin/iptables -w -nL FORWARD'
+- name: list iptables rules filtering chain
+  command: '/usr/sbin/iptables -w -nL OPENSHIFT-OUTBOUND-FILTERING'
   changed_when: False
-  register: forwardchain
+  register: filteringchain
+  ignore_errors: True
 
 - name: list iptables rules logging chain
   command: '/usr/sbin/iptables -w -nL oo-outbound-logging'
   changed_when: False
   register: loggingchain
   ignore_errors: True
 
-- name: add in-memory iptables chain
+- name: add in-memory filtering chain
+  command: "/usr/sbin/iptables -N OPENSHIFT-OUTBOUND-FILTERING"
+  when: not "OPENSHIFT-OUTBOUND-FILTERING" in filteringchain.stdout
+  with_items: "{{ outbound_logging_chain }}"
+
+- name: add in-memory logging chain
   command: "/usr/sbin/iptables -N oo-outbound-logging"
   when: not "oo-outbound-logging" in loggingchain.stdout
   with_items: "{{ outbound_logging_chain }}"
 
 - name: modify in-memory iptables rules
   command: "/usr/sbin/iptables {{ item.line }}"
-  when: not item.regex in forwardchain.stdout and not item.regex in loggingchain.stdout
+  when: not item.regex in filteringchain.stdout and not item.regex in loggingchain.stdout
```

Comment:
even though this works, I don't like the idea of a regex being treated as a string. What if somebody comes along later and updates the "regex" which only supports a fixed-string? I suggest something like this instead:

`when: not (filteringchain.stdout | search(item.regex) and not (loggingchain.stdout | search(item.regex)`